src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -178,16 +241,12 @@ static CPythonEvaluator()
             }
 
             InstallPython();
-
             if (!PythonEngine.IsInitialized)
             {
                 PythonEngine.Initialize();
                 PythonEngine.BeginAllowThreads();
+                
             }
-
-            IntPtr gs = PythonEngine.AcquireLock();
-            try
-            {
```

Comment:
this lock stuff is not needed anymore ?

Reply:
nope, I believe this is the same as using Py.GIL() - see the description and threading wiki that says as much.